features/FEATURE_BLE/ble/gap/Gap.h

```diff
@@ -724,9 +731,9 @@ class Gap {
      * @version 5+
      */
     virtual bool isPeriodicAdvertisingActive(advertising_handle_t handle);
-
+#endif // BLE_FEATURE_PERIODIC_ADVERTISING
     /*                                     scanning                                              */
-
+#if BLE_ROLE_OBSERVER
```

Comment:
Extended Advertising is mandatory for some of the features present in that block.

Reply:
which? the feature is checked at runtime in the implementation

Reply:
extended now guarded with ifdef